src/consts.h

```diff
@@ -46,6 +47,17 @@ typedef enum {
     TS_AGG_TYPES_MAX // 13
 } TS_AGG_TYPES_T;
 
+
+typedef enum DuplicatePolicy {
+    DP_INVALID = -1,
+    DP_NONE = 0,
+    DP_BLOCK,
```

Comment:
@danni-m I think you should set the values for all the enums since we persist them and if someone will add or remove any in the middle it will all break

Reply:
good idea